### PR TITLE
263 update icon component

### PR DESCRIPTION
**Pull request by vlacher12**

### CODE REVIEW

File: src/components/accordion/primary/index.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ export const AccordionPrimary = React.forwardRef((props: AccordionProps, ref: an
     onIconClick(e);
   };
 
-  const checkTypeIconRight = typeof iconRight === 'boolean';
-  const checkTypeIconLeft = typeof iconLeft === 'boolean';
+  // const checkTypeIconRight = typeof iconRight === 'boolean';
```
Comment (lazzystep) [inline src/components/accordion/primary/index.tsx:35]: do we need it? maybe better to remove it?

**Review by lazzystep — APPROVED** [review]

File: src/components/accordion/primary/index.tsx (hunk 1 above)
Comment (lazzystep) [reply src/components/accordion/primary/index.tsx:35]: same for all commented lines

**Comment by vlacher12** [comment]
image
What linaria-cache is doing here?)

**Review by lazzystep — APPROVED** [review]

File: src/components/responsive-table/columns.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const Columns = ({ columns, sort, handleSort, sortIcons }: ColumnsProps): JSX.El
         minWidth={minWidth}
         sortName={sortName}
         sort={sort}
-        sortIcons={sortIcons}
+        // sortIcons={sortIcons}
```
Comment (kkurylchenko) [inline src/components/responsive-table/columns.tsx:21]: comment

File: src/components/select/select-component.tsx [hunk 3]
```diff
@@ -8,22 +8,22 @@ import { Icon, Text } from 'components';
...
+  // };
 
   /* eslint-disable-next-line no-underscore-dangle */
-  const deleteButtonIsShown = onDeleteItem && !data.__isNew__ && !isSelected;
+  // const deleteButtonIsShown = onDeleteItem && !data.__isNew__ && !isSelected;
 
   return (
     <components.Option {...props}>
-      {isSelected && <Icon icon={Icon.icons.check} className='icon' />}
+      {/* {isSelected && <Icon icon='check' className='icon' />} */}
```
Comment (kkurylchenko) [inline src/components/select/select-component.tsx:24]: comment

**Review by kkurylchenko — APPROVED** [review]

**Comment by vlacher12** [comment]
@sergdeev Ready to go?

**Pull request merged**